internal/pkg/pct/news.go

```diff
@@ -0,0 +1,89 @@
+package pct
+
+import (
+	"bytes"
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/fatih/color"
+	"github.com/rodaine/table"
+)
+
+// The top level structure
+type RSS struct {
+	Item []Item `xml:"channel>item" json:"news"`
+}
+
+// The structure of the information we wish to retrieve
+type Item struct {
+	Title string `xml:"title" json:"title"`
+	Link  string `xml:"link" json:"link"`
+}
+
+func News(Url string, Format string) error {
+	fmt.Printf("Retrieving HTML code of %s ...\n", Url)
+	// Retrieve the html of the page
+	resp, err := http.Get(Url) //#nosec G107
+	if err != nil {
+		return err
+	}
+	// do this now so it won't be forgotten
+	defer resp.Body.Close()
+	// reads html as a slice of bytes
+	html, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	// Create a variable of the top structure type set out above
+	var items RSS
+
+	// Unmarshal the xml
+	err = xml.Unmarshal([]byte(html), &items)
+	if err != nil {
+		return fmt.Errorf("Error ith retrieved yaml")
```

Comment:
Don't create error text when you have an error from a library. Always return the library error at a minimum
```suggestion
		return err
```

Reply:
err is nor returned